Cargo.toml

```diff
@@ -18,18 +18,20 @@ include = [
     "/LICENSE-MIT",
     "/LICENSE-APACHE",
 ]
-edition = "2018"
+edition = "2021"
 
 [badges]
 coveralls = { repository = "eldruin/ads1x1x-rs", branch = "master", service = "github" }
 
 [dependencies]
 nb = "1"
-embedded-hal = { version = "0.2.2", features = ["unproven"] }
+embedded-hal = "1"
 
 [dev-dependencies]
-linux-embedded-hal = "0.3"
-embedded-hal-mock = "0.7"
+embedded-hal-mock = "0.10"
+
+[target.'cfg(target_os = "linux")'.dev-dependencies]
```

Comment:
The linux examples will still not build, though. What were you trying to fix with this change?

Reply:
This is so that at least `cargo test --lib --tests` works locally on macOS. Otherwise, it will try to build a Linux-only dependency which is only needed for examples.